### ISSUE TITLE
growedition, how can i?

### ISSUE THREAD

**Issue by srn87** [HashLips/hashlips_art_engine#1224]
hey guys someone knows if i have 2k nfts done and i want to starts again i would like after this, if is possible?

**Comment by bolshoytoster** [comment]
@srn87 do you mean you want to continue a collection or start a new one starting from 2k?

**Comment by srn87** [comment]
@bolshoytoster I would like to continue the collection but where did the previous print run end, Thanks in advance

**Comment by bolshoytoster** [comment]
@srn87 you can use [my fork](https://github.com/bolshoytoster/hashlips_art_engine) and set `growEditionSizeTo` to whatever you want the full collection to be.

**Comment by srn87** [comment]
@bolshoytoster maybe I explained myself wrong, I have 2000 nfts already created but I don't want to launch node index.js again because it overwrites the previous ones, so I want it to start in 2001 and start saving from there, thankss

**Comment by bolshoytoster** [comment]
@srn87 yes, my fork doesn't overwrite the existing collection.

**Comment by srn87** [comment]
@bolshoytoster oks, i copy then, thanks a lot

**Comment by srn87** [comment]
hi @bolshoytoster  i do with your fork, i receive this how can i do now for do more? do you have some var  like "starteditionfrom" = 10 for example?
![2022-06-04](https://user-images.githubusercontent.com/66142800/171996598-a7cabdf3-d50f-4c07-a8f9-9697661653f7.png)

**Comment by bolshoytoster** [comment]
@srn87 yes, there is `startEditionFrom`, the same way you set `growEditionSizeTo`.

**Comment by srn87** [comment]
thanks @bolshoytoster work it, could you know how can i add in this metadata var; 
description something like this, i want to put the name of layer in description, would be something there:
description ="This is the " + `${namePrefix}` + " of the nft collection. "
But need name of layer concatenated to namePrefix, so that it says which version it is in the description, for example if it is drawing 5.png say: This is the "nameprefix"#5 of the nft collection
